project_purchase_line_link/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Purchase Line Link",
+    "summary": """
+        Add button in Proyects that joins them with the purchase order lines 
+        that have the same analytic account as the project.
+        Only to purchase users.

```

Comment:
Typo fixed and more adjusted:
```suggestion
        Add button in Projects that joins them with the purchase order lines 
        that have the same analytic account as the project.
```